concordia/templates/transcriptions/item_detail.html

```diff
@@ -51,9 +51,11 @@ <h1 class="m-3">{{ item.title }}</h1>
     </div>
     <div class="row">
         <form method="get" class="w-50 mx-auto">
-            <h6>Filter Assets:</h6>
             <div class="form-group row">
                 <div class="input-group input-group-sm">
+                    <div class="input-group-prepend">
+                        <span class="input-group-text" id="inputGroup-sizing-sm">Filter:</span>
```

Comment:
We're not using this ID, are we?

```suggestion
                        <span class="input-group-text">Filter:</span>
```